src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -63,6 +63,10 @@ final class OrderedClassElementsFixer extends AbstractFixer implements Configura
         'method_public_static' => ['method_static', 'method_public'],
         'method_protected_static' => ['method_static', 'method_protected'],
         'method_private_static' => ['method_static', 'method_private'],
+        'method_public_abstract' => ['method_abstract', 'method_public'],
+        'method_protected_abstract' => ['method_abstract', 'method_protected'],
+        'method_public_abstract_static' => ['method_abstract_static', 'method_abstract', 'method_public'],
+        'method_protected_abstract_static' => ['method_abstract_static', 'method_abstract', 'method_protected'],
```

Comment:
Shouldn't this be `method_abstract_protected_static` to follow PSR12?

Reply:
changed